doc/setting_up_sidewalk_environment/setting_up_hardware.rst

```diff
@@ -88,6 +88,77 @@ Connect the Semtech shield to the Arduino-compatible headers on the development
     +---------------+-------------+
     | NRESET        | P0.04       |
     +---------------+-------------+
+   
+   .. tab:: nRF54L15 PDK
+
+      The nRF54L15 PDK does not have the Arduino-compatible header.
+      The connection layout may differ depending on the revision of your PDK.
+      GPIO assignments are recommended by the Nordic Semiconductor. 
+      However, if you need to modify the pinout, refer to the `Infocenter`_ SoC documentation, as some functions are limited to dedicated pins. 
+
+      The nRF54L15 PDK operates on lower voltage level (1.8V). 
+      The Semtech Shield is compatible with this voltage and can be safely provided to the pin marked as 3.3V.
+      Due to the lower voltage, the Semtech radio cannot achieve the full transmit power, and it will reach a plateau at 15 dBm gain even when a higher value is configured.
+      The specification of the Semtech module describes that it is possible to provide separate, higher voltage source for radio, while maintaining lower voltage for communication. 
+      However, the recommended shield does not support this configuration.
+
+      .. tabs::
+
+         .. tab:: PDK revision v0.2.1
+
+            The v0.2.1 PDK revision does not require any hardware modification, but the buttons are reused to connect the Semtech SX126X shield.
+            Pressing the buttons while working with Semtech module can lead to malfunctions and undefined behavior.
+            This issue is caused by the limitation of the v0.2.1 PDK revision. 
+            It is possible to assign different pins for the radio shield, but you might need to disable some of the PDK's functions, or modify connections by cutting or soldering PDK bridges.
+
+            +---------------+-------------+
+            | SX126X shield | nRF54L15 PDK |
+            +===============+=============+
+            | BUSY          | P1.12       |
+            +---------------+-------------+
+            | DIO1          | P1.10       |
+            +---------------+-------------+
+            | NSS           | P2.10       |
+            +---------------+-------------+
+            | ANT_SW        | P2.07       |
+            +---------------+-------------+
+            | SPI MOSI      | P2.08       |
+            +---------------+-------------+
+            | SPI MISO      | P2.09       |
+            +---------------+-------------+
+            | SPI SCK       | P2.06       |
+            +---------------+-------------+
+            | NRESET        | P1.11       |
+            +---------------+-------------+
```

Comment:
```suggestion
            +---------------+--------------+
            | SX126X shield | nRF54L15 PDK |
            +===============+==============+
            | BUSY          | P1.12        |
            +---------------+--------------+
            | DIO1          | P1.10        |
            +---------------+--------------+
            | NSS           | P2.10        |
            +---------------+--------------+
            | ANT_SW        | P2.07        |
            +---------------+--------------+
            | SPI MOSI      | P2.08        |
            +---------------+--------------+
            | SPI MISO      | P2.09        |
            +---------------+--------------+
            | SPI SCK       | P2.06        |
            +---------------+--------------+
            | NRESET        | P1.11        |
            +---------------+--------------+
```

the same needs to be done with the other table (if the number of characters isn't aligned the table won't render)

Reply:
ok